leetcode/src/2279.c

```diff
@@ -0,0 +1,29 @@
+int compare(const int* i, const int* j)
+{
+    return *i - *j;
+}
+
+// Sorting.
+// Runtime: O(n*log(n))
+// Space: O(n)
+int maximumBags(int* capacity, int capacitySize, int* rocks, int rocksSize, int additionalRocks) {
+    int* capacityLeft = malloc(capacitySize * sizeof(int));
+    for (int i = 0; i < capacitySize; i++){

```

Comment:
Not sure if you want to do the same here.
```suggestion
    for (int i = 0; i < capacitySize; i++) {
```